client/src/pages/HomePage.jsx

```diff
@@ -1,8 +1,45 @@
-import React from "react";
-import '../demo.less';
+import React from 'react';
+import { Col, Row } from 'antd';
+
+import '../styles/homepage.less';
+import IngredientIcon from '../assets/ingredient-icon.png';
+import RecipeIcon from '../assets/recipe-icon.png';
+import SelectionCard from '../components/SelectionCard';
 
 const HomePage = () => {
-  return <div className={"demo-less"}>This is the homepage</div>;
-};
 
+  return (
+    <>
+      <Row align='middle' className={'homepageRoot'}>
+        <Col span={16} offset={4}>
+          <SelectionCard
+            selectionTitle='Search By Ingredients'
+            coverImage={
+              <div className={'iconsWrapper'}>
+                <img
+                  className={'icons'}
+                  alt='ingredient-icon'
+                  src={IngredientIcon}
+                />
+              </div>
+            }
+            routePath={'/ingredient'}
+          />
+        </Col>
+        <Col span={16} offset={4}>
+          <SelectionCard
+            className={'cardWrapper'}
```

Comment:
Is this a class you'll be making or was it made in another commit?

Reply:
It's a custom component under the `/component` folder. I figured the recipe selection page can reuse these cards anyway so why not make it a generic component

Reply:
Sorry, are you talking about `SelectionCard` or `cardWrapper`?

Reply:
ohhh sorry mb mb. The `cardWrapper` class is defined in the `homepage.less` file.

Reply:
oh wait nvm ... I messed up lol I was playing around with the classes and forgot to purge this one. I'll remove it

Reply:
fixed

Reply:
I think you put it as `iconsWrapper` in `homepage.less`, either way, it's a really minor nitpick on my end

Reply:
yea I think I was trying to create a class to wrap the antd card but didn't end up working so I just removed the entire class lol. Must've missed this one. Thanks for catching it :)